sdk/src/Core/AWSSDK.Core.NetStandard.csproj

```diff
@@ -83,6 +83,7 @@
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
     <PackageReference Include="System.Buffers" Version="4.5.1" />
     <PackageReference Include="System.Memory" Version="4.5.5" />
-    <PackageReference Include="System.Text.Json" Version="8.0.5" />
+    <PackageReference Include="System.Text.Json" Version="6.0.11" />
```

Comment:
Can you add a comment here explaining why we are locking to 6.0.11 which will be different then the nuspec file? This seems like something we will forget why in a few years from now.